swupd/data_test.go

```diff
@@ -5,29 +5,6 @@ import (
 	"testing"
 )
 
-func TestReadServerINI(t *testing.T) {
```

Comment:
You remove this test here because the `swupd` package doesn't read the config anymore. Can you add a similar one to where the config is actually being read?

Reply:
What do you mean? To create a new test file for config module? I removed this test because what it tests does not exists anymore.